gazelle/def.bzl

```diff
@@ -16,6 +16,4 @@
 """
 
 GAZELLE_PYTHON_RUNTIME_DEPS = [
```

Comment:
Please update the `example` docs and the `gazelle/README.md` to not use this import.